specs/commandline/verify.md

```diff
@@ -37,9 +37,11 @@ Usage:
 Flags:
   -d,  --debug                       debug mode
   -h,  --help                        help for verify
+       --oci-layout                  [Preview] whether artifact is OCI image layout
```

Comment:
```suggestion
       --oci-layout                 [Preview] use OCI image layout for storage
```

Reply:
Suggestion: --oci-layout   [Preview] Verify the artifact stored in the OCI image layout